adsplanetnamepipe/utils/common.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import regex
+import unicodedata
+from typing import List, Dict, Union
+
+from adsputils import setup_logging, load_config
+
+logger = setup_logging('utils')
+config = {}
+config.update(load_config())
+
+class EntityArgs():
+    """
+    EntityArgs is a class to encapsulate the planetry name entity information
+    """
+    target: str
+    feature_type: str
+    feature_type_plural: str
+    feature_name: str
+    ambiguous_feature_name_context: List[str]
+    multi_token_containing_feature_names: List[str]
+    name_entity_labels: List[Dict[str, Union[str, int]]]
+    all_targets: List[str]
+
+    def __init__(self, target: str, feature_type: str, feature_type_plural: str, feature_name: str,
+                 context_ambiguous_feature_names: List[str], multi_token_containing_feature_names: List[str],
+                 name_entity_labels: List[Dict[str, Union[str, int]]], all_targets: List[str]=[]):
+        """
+
+        :param target:
+        :param feature_type:
+        :param feature_type_plural:
+        :param feature_name:
+        :param context_ambiguous_feature_names: list of context for feature name, can be empty
+        :param multi_token_containing_feature_names: list of multi token feature names that contains the feature name, can be emtpy
+        """
+        self.target = target
+        self.feature_type = feature_type
+        self.feature_type_plural = feature_type_plural
+        self.feature_name = feature_name
+        self.context_ambiguous_feature_names = context_ambiguous_feature_names
+        self.multi_token_containing_feature_names = multi_token_containing_feature_names
+        self.name_entity_labels = name_entity_labels
+        self.all_targets = all_targets
+
+
+class Synonyms(object):
+
+    # generated 5/20/2022
+    synonym_list = {
```

Comment:
Need to ask, I dont know the answer.

Reply:
I asked Alberto about this, and he said that he talked to you and Taylor. I brought up the same issue on the side of resolver with Taylor. Over there, I am one step closer. There are files on the server that Edwin modifies on a regular basis. They are called source matchers. Reading and processing them takes some time, so I do that offline, create a serializable object, and then load that every time the service is deployed. I actually went through the motion, got files from the server, created a serializable object, uploaded it to GitHub, and asked Taylor to see if he could upload the serializable object to the pod without making a release. It was just before his vacation. He said he would think about it afterward. So, I think I should do the same thing here as well: make a serializable object and update it on a regular basis. I will make an issue out of this and wait to see if it works for the resolver. If it does, then I’ll proceed this way; otherwise, I’ll go with another approach.

created the issue https://github.com/adsabs/ADSPlanetaryNamesPipeline/issues/2